pkg/ip/Makefile

```diff
@@ -0,0 +1,3 @@
+IMAGE=ip
```

Comment:
Maybe it's better to call this `netcfg` or similar? Don't have a particularly strong view on this, though.